### ISSUE TITLE
Regression with explicitly defined fill colours

### ISSUE THREAD

**Issue by RogerSelwyn** [alexarch21/history-explorer-card#62]
The following no longer results in a fill with no line for sensor.nest_heating, instead a line is generated. Reverting to 1.0.23 solves the problem. 1.0.24 introduces the problem.

```
- type: line
    options:
      ymin: 0
    entities:
      - entity: sensor.nest_current_temperature
        lineMode: curves
        name: Current
        color: '#84bb7199'
      - entity: sensor.nest_set_temperature
        lineMode: stepped
        name: Set
        color: '#6592eb'
      - entity: sensor.nest_heating
        lineMode: stepped
        name: Heating
        fill: '#84bb7199
```

**Comment by alexarch21** [comment]
Confirmed, regression in 1.0.26 due to the auto colors update. Will take at least two weeks for me to be able to fix this, as I'm traveling right now and don't have access to my dev system.

Temporary workaround is to downgrade to 1.0.25.

**Comment by RogerSelwyn** [comment]
No problem. I’ll try 1.0.25. No rush.

**Comment by alexarch21** [comment]
Fixed in 1.0.27.
